### ISSUE TITLE
fetchAppDefinition raises exception on initial login

### ISSUE THREAD

**Issue by mixmixmix** [road86/bahis-desk#31]
Somehow previous PR (https://github.com/road86/bahis-desk/commit/f4f65fa42ac8d10bb978dffd2dd6cb8416dcd8f5) introduced this exception that remains in our draft PR https://github.com/road86/bahis-desk/pull/30 and might cause some of the issues we are experiencing. 
It isn't called on repeat login so it might be causing https://github.com/road86/bahis-desk/issues/28

https://github.com/road86/bahis-desk/blob/4a0b28f51ef1a443bb93e444c51054a588656cb3/public/electron.js#L197
```
Exception has occurred: TypeError: Cannot read property 'definition' of undefined
  at IpcMainImpl.fetchAppDefinition (/Users/mix/repos/bahis-desk/public/electron.js:197:86)
    at IpcMainImpl.emit (events.js:315:20)
    at Object.<anonymous> (electron/js2c/browser_init.js:161:10307)
    at Object.emit (events.js:315:20)
    at Object.callbackTrampoline (internal/async_hooks.js:131:14)
```

**Comment by ChasNelson1990** [comment]
Ah yes, I did notice this... _but_ I am not 100% sure what that table is doing and why it exists?

**Comment by mixmixmix** [comment]
yes, me neither...

**Comment by ChasNelson1990** [comment]
closed by #30
